bin/detail/call.py

```diff
@@ -63,15 +63,16 @@ def call(call_args, logging, cache_file='', ignore=False, sleep=0):
   for i in call_args:
     pretty += '  `{}`\n'.format(i)
   pretty += ']\n'
-  print(pretty)
+  if logging.verbosity == 'full':
```

Comment:
Just tried it. Looks kind of strange to me, I see some CMake configuration output, then some compile commands but don't see what command originally executed.